ui/app/styles/components/allocation-fs.scss

```diff
@@ -0,0 +1,70 @@
+.fs-explorer.table {
```

Comment:
Can you rename this file to `fs-explorer.scss`? I'm trying to keep a filename === class name convention for css components.